### PR TITLE
add missing stub in av.sidedata

### PR DESCRIPTION
**Pull request by laggykiller**
Requires: #1306

### CODE REVIEW

File: av/sidedata/motionvectors.pyi [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from typing import Any, Sequence
+
+import numpy as np
+
+from .sidedata import SideData
+
+class _MotionVector(SideData):
```
Comment (WyattBlue) [inline av/sidedata/motionvectors.pyi:7]: _MotionVector is not supposed to be part of the Public API, so it's attributes should be merged with MotionVector

File: av/sidedata/sidedata.pyi [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    SPHERICAL: int
+    CONTENT_LIGHT_LEVEL: int
+    ICC_PROFILE: int
+    SEI_UNREGISTERED: int
+
+class SideData(Buffer):
+    type: Type
+    DISPLAYMATRIX: int
+
+class _SideDataContainer:
```
Comment (WyattBlue) [inline av/sidedata/sidedata.pyi:30]: Same applies here

**Comment by laggykiller** [comment]
Please check, I am not sure if this fix is correct...

**Comment by WyattBlue** [comment]
I think #1311 needs to be merged before this can.

**Pull request merged**